### ISSUE TITLE
Ninja shoes cover eyes?

### ISSUE THREAD

**Issue by palpatine213** [tgstation/-tg-station#8458]
I have no idea what is going on here, but when doing surgery on someone's eyes, if they are wearing the ninja suit shoes, it will say that their eyes need to be exposed first.

Steps to reproduce:
Strip completely
spawn and equip a pair of ninja suit shoes
use the surgical drapes to start an eye surgery
be confused as to why it says the eyes are covered.

**Comment by RemieRichards** [comment]
They protect the exact same places regular shoes do.

**Comment by palpatine213** [comment]
Since when did you need to remove your shoes to get eye surgery? I tested this with regular shoes, and it only seemed to happen with ninja shoes.

**Comment by RemieRichards** [comment]
What can I say, they protect the exact same places regular shoes do, and only those.

**Comment by phil235** [comment]
The surgery proc that check if the location is accessible currently look at all clothing items and put all flags stuff together, and since NOSLIP and GLASSESCOVERSEYES share the same value of 1024, it means the proc thinks the eyes are covered when wearing any items with NOSLIP flag.

**Comment by Miauw62** [comment]
kek level: top

**Comment by RemieRichards** [comment]
Still an issue, Ideally the flags bitflag gets changed to a list so we can use infinite many numbers, since we're already over the max amount of flags in a bit field for the flags var.

**Comment by Miauw62** [comment]
the point of using bitflags is that theyre fast, lists are slow af

**Comment by RemieRichards** [comment]
@Miauw62 the speed is worthless when we have ridiculously issues like this and have to tippy toe around each use of flags for each atom, since some atoms only ever use a few of the flags but if they ever used a different one they'd magically break, just like now where we have flags overlapping (there's another 1-2 flags that overlap besides the 2 at fault here)

tl;dr the speed impact (if any at all) is more than acceptable for a working game and safe environment for using flags.

**Comment by Miauw62** [comment]
alternatively, you could introduce a new bitflag.
the issues here are caused by people trying to compress flags into the same variables when this is really not neccesary.
